src/include/daos_srv/ras.h

```diff
@@ -88,8 +88,8 @@ ras_event_sev_enum_to_name(enum ras_event_sev severity)
 }
 
 enum ras_event_type {
-	RAS_TYPE_RANK_STATE_CHANGE	= 0,
-	RAS_TYPE_INFO_ONLY,
+	RAS_TYPE_RANK_STATE_CHANGE	= (1 << 0),
+	RAS_TYPE_INFO_ONLY		= (1 << 1),
```

Comment:
Hmm. So this is kinda cool, but it occurs to me that we might be limiting the number of possible event ids and types, right? Some quick googling tells me that an enum is only guaranteed to be large enough to hold an int value, and this is of course architecture-dependent but is usually only 4 bytes.

Reply:
will revert